feathr-example/static_features.ipynb

```diff
@@ -0,0 +1,1632 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Feathr Feature Store on Home Credit\n",
+    "\n",
+    "This notebook illustrates the use of Feature Store to create a model for home credits. It includes these steps:\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisite: Install Feathr\n",
+    "\n",
+    "Install Feathr using pip:\n",
+    "\n",
+    "`pip install -U feathr pandavro scikit-learn`\n",
+    "\n",
+    "Or if you want to use the latest Feathr code from GitHub:\n",
+    "\n",
+    "`pip install -I git+https://github.com/linkedin/feathr.git#subdirectory=feathr_project pandavro scikit-learn`"
```

Comment:
Maybe point to a release version?

Reply:
This one is stating if you want to use the latest version of feathr thus it getting directly from the github repo and would install what is in the `main` branch. 

Changed the prior statement to this to install the specific `v0.7.2` release.